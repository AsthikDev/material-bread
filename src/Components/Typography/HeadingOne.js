import React, { Component } from 'react';
import PropTypes from 'prop-types';
import { StyleSheet } from 'react-native';
import BaseText from './BaseText';

const styles = StyleSheet.create({
  headingOne: {
    fontWeight: '300',
    fontSize: 96,
    lineHeight: 96,
    letterSpacing: -1.5,
  },
});

export default class HeadingOne extends Component {
  static propTypes = {
    children: PropTypes.node,
  };

  render() {
    return (
      <BaseText typographyStyles={styles.headingOne} {...this.props}>
        {this.props.children}
      </BaseText>
    );
  }
}
