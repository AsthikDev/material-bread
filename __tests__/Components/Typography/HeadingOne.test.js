import React from 'react';
import HeadingOne from '../../../src/Components/Typography/HeadingOne';

import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const tree = renderer.create(<HeadingOne />).toJSON();
  expect(tree).toMatchSnapshot();
});
