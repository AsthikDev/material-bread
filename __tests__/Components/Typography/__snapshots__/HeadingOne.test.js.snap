// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Text
  style={
    Object {
      "color": "rgba(0, 0, 0, 0.87)",
      "fontSize": 96,
      "fontWeight": "300",
      "letterSpacing": -1.5,
      "lineHeight": 96,
    }
  }
  typographyStyles={
    Object {
      "fontSize": 96,
      "fontWeight": "300",
      "letterSpacing": -1.5,
      "lineHeight": 96,
    }
  }
/>
`;
